lib/libvfio-user.c

```diff
@@ -1334,7 +1335,7 @@ vfu_create_ctx(vfu_trans_t trans, const char *path, int flags, void *pvt,
     vfu_ctx->uuid = strdup(path);
     if (vfu_ctx->uuid == NULL) {
         err = errno;
```

Comment:
`-errno` (since you're touching this code)

Reply:
Ack